From a754e9cf29415d110c4fe6fb1fe0d060d765a73e Mon Sep 17 00:00:00 2001
From: Michel Machado <michel@digirati.com.br>
Date: Thu, 19 May 2016 08:39:36 -0400
Subject: [PATCH] f3read: add "#include <limits.h>" to f3read.c

This addresses a compiling issue on Macs.

@jksinton found and solved this problem:
https://github.com/AltraMayor/f3/issues/44
---
 f3read.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/f3read.c b/f3read.c
index 0842071..c5c20d2 100644
--- a/f3read.c
+++ b/f3read.c
@@ -5,6 +5,7 @@
 #include <stdint.h>
 #include <inttypes.h>
 #include <stdio.h>
+#include <limits.h>
 #include <string.h>
 #include <stdlib.h>
 #include <errno.h>
